README.rst

```diff
@@ -285,6 +285,18 @@ If you want to compile the applications under ``examples/``, you need
 to remove or rename the ``event.h`` from libev's installation, because
 it conflicts with libevent's installation.
 
+Notes for installation on Linux systems
+--------------------------------------------
+After installing nghttp2 tool suite with `make install` one might experience a similar error:
+
+```
+nghttpx: error while loading shared libraries: libnghttp2.so.14: cannot open shared object file: No such file or directory
+```
```

Comment:
This reST formatted document, and the verbatim text is achieved using ".. code-block:: text" like this:

```
.. code-block:: text

    nghttpx: error while loading shared libraries: libnghttp2.so.14: cannot open shared object file: No such file or directory
```